sound/soc/sof/ipc4-mtrace.c

```diff
@@ -553,6 +553,12 @@ static int ipc4_mtrace_init(struct snd_sof_dev *sdev)
 	if (!priv)
 		return -ENOMEM;
 
+	priv->state_info.logs_priorities_mask = devm_kzalloc(sdev->dev,
+				sizeof(u32) * ipc4_data->max_libs_count,
+			    GFP_KERNEL);
```

Comment:
devm_kcalloc() ?

Reply:
sure, I tried to find this function but was failed. Thanks!